.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  release:
+    types: [released, prereleased]
+  workflow_dispatch:  # allow manually running from the Actions tab
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      BUILDX_PLATFORMS: linux/amd64,linux/arm64,linux/arm/v7
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
these two you would have to add to https://github.com/stevearc/pypicloud-docker/settings/secrets/actions

which is safe as only you can trigger this workflow